lib/resty/etcd/v3.lua

```diff
@@ -676,6 +679,7 @@ local function request_chunk(self, method, path, opts, timeout)
         body    = body,
         query   = query,
         headers = headers,
+        ssl_server_name = self.sni,
```

Comment:
Should we update the `http_cli:request_uri` part?

Reply:
here: https://github.com/api7/lua-resty-etcd/pull/145/files#diff-f87343d4dad124395f327f1b3a570e863a29ac213f9385d80b75b59bf20ff33cR91?